lib/media/track/remotetrackpublication.js

```diff
@@ -66,15 +66,28 @@ class RemoteTrackPublication extends TrackPublication {
       }
     });
 
+    let error = null;
+    let isEnabled;
+    let isSwitchedOff = false;
```

Comment:
@makarandp0 Nitpick: add a newline between the variable declarations and setting the "updated" event listener?